package.json

```diff
@@ -31,10 +31,11 @@
     "release": "node ./lib/cli.js publish -y",
     "release:docs": "yarn docs:build && yarn gh-pages -d docs/.vuepress/dist --dotfiles",
     "start": "tsc -w --preserveWatchOutput",
-    "test": "yarn test:unit && yarn test:func && yarn test:e2e",
-    "test:e2e": "jest --config jest.e2e.js",
-    "test:func": "jest --config jest.func.js",
-    "test:unit": "jest",
+    "test": "jest",
+    "test:all": "yarn test:unit && yarn test:func && yarn test:e2e",
```

Comment:
Can test:all not be just "jest" now too? Or do they have to STILL run in sequence?

Reply:
They don't strictly have to run in sequence, it's just easier to find the actual error that way. So you can run `yarn test` to run everything in whatever order jest decides, or `yarn test:all` to run in a friendlier order.